az-kafka-dependency-plugin/src/main/Java/trigger.kafka/Test.java

```diff
@@ -0,0 +1,61 @@
+package trigger.kafka;
+
+import azkaban.flowtrigger.DependencyInstanceCallback;
+import azkaban.flowtrigger.DependencyInstanceConfig;
+import azkaban.flowtrigger.DependencyInstanceConfigImpl;
+import azkaban.flowtrigger.DependencyInstanceContext;
+import azkaban.flowtrigger.DependencyInstanceRuntimeProps;
+import azkaban.flowtrigger.DependencyInstanceRuntimePropsImpl;
+import azkaban.flowtrigger.DependencyPluginConfig;
+import azkaban.flowtrigger.DependencyPluginConfigImpl;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import trigger.kafka.Constants.DependencyInstanceConfigKey;
+import trigger.kafka.Constants.DependencyPluginConfigKey;
+
+
+public class Test {
```

Comment:
what is this class for?

Reply:
please move this class and KafkaProducerTest to another package and document the purpose.